### PR TITLE
fix #74503 hotcleaner.com

### LINKED ISSUE #74503: hotcleaner.com
### Issue URL (Ads)
 [https://www.hotcleaner.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.hotcleaner.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 nightly 15
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180), <br>Block Push API, <br>Block Location API
DNS filtering: | server: `https://dns-unfiltered.adguard.com/dns-query`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List
Custom filter subscriptions: | https://raw.githubusercontent.com/AdguardTeam/FiltersRegistry/master/filters/filter_15_DnsFilter/filter.txt

### PR DESCRIPTION
**Pull request by wout-codes**
#74503

### CODE REVIEW

**Comment by wout-codes** [comment]
@zloyden Any ideas on how to block Ad leftover and cookie at this page `www.hotcleaner.com/cookie-editor`?

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -545,6 +545,9 @@ kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74503
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=hotcleaner.com
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=hotcleaner.com
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:550]: I don't see adb detection here.

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:550]: That rule kills this detection (failed to load resource) there are still a lot of placeholders and advertisement name and a cookie in this domain `https://www.hotcleaner.com/cookie-editor/`
`image`

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:550]: Nevermind you fix it. By the way, you missed this, 
<details><summary>Screenshot:</summary>
image
</details><br/>

Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt:550]: Please use the spoiler for the screenshot.
It was fixed.

Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt:550]: > By the way, you missed this

Where?
<details><summary>Screenshot:</summary>

image

</details><br/>

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:550]: I still get it though

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:550]: @zloyden Wait for 10 secs

Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt:550]: Well, wait for the filters update in 2-3 hours.

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:550]: @zloyden I copied all the rules you added in that commit then navigated to the site.

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:550]: @zloyden I can reproduce this issue with only AdGuard App. With the AdGuard Browser Ad-Blocker it's blocked.

Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt:550]: Strange, I can reproduce it now. Fixed, thanks for the participation.

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:550]: Why is the app like this though?

**Pull request closed without merging**